installEmailColumns.ps1

```diff
@@ -41,7 +41,7 @@ Try {
 
     #Download xml provisioning template
     $WebClient = New-Object System.Net.WebClient   
-    $Url = "https://raw.githubusercontent.com/OnePlaceSolutions/EmailColumnsPnP/master/email-columns.xml"    
```

Comment:
Should we also delete this file from the repository?

Reply:
Happy to do so, will update these PR's.

Reply:
Actually it's just this PR that needs to be updated, the other refers to this URL and doesn't store a copy of it